### PR TITLE
Revision of Workshop for FOSSGIS 2016

### PR DESCRIPTION
**Pull request by ahennr**
Title says it all.

In this revision I focused on chapters covering basics, REST and GWC.

Please review @buehner and @dnlkoch .

### CODE REVIEW

**Comment by dnlkoch** [comment]
Great!

File: src/basics/README.md [hunk 1]
```diff
@@ -15,15 +15,16 @@ Die beiden Links verweisen auf die Dokumentationen der letzten stabilen Version.
...
 
 Üblicherweise wird der GeoServer für einen Produktivbetrieb als (Java-)Standalone-Servlet
-in Form einer .war-Datei bereitgestellt, welche unter <http://geoserver.org/download/>
-heruntergeladen werden kann. Die .war-Datei muss anschließend auf einem
-Servlet-Container, z.B. [Tomcat](http://tomcat.apache.org/) oder
-[Jetty](http://eclipse.org/jetty/) veröffentlicht, d.h. *deployed* werden. Anschließend
+in Form einer `.war` - Datei bereitgestellt, welche unter <http://geoserver.org/download/>
+heruntergeladen werden kann. Die `.war` - Datei muss anschließend auf einem
+Servlet-Container (bspw. [Tomcat](http://tomcat.apache.org/) oder
+[Jetty](http://eclipse.org/jetty/)) veröffentlicht werden (häufig auch *deploy* genant). Anschließend
```
Comment (annarieger) [inline src/basics/README.md]: Typo -> `genannt`

File: src/basics/installextensions.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ Daten aus bestimmten SQL-Datenbanken beziehen (z.B. MySQL oder Oracle). Ebenso
 gibt es Erweiterungen, die es ermöglichen *Excel* (und weitere) als Ausgabeformat
 zu unterstützen.
 
+Eine vergleichsweise neues Zusatzmodul ist die [Importer-Extension](http://docs.geoserver.org/maintain/en/user/extensions/importer/using.html).
```
Comment (annarieger) [inline src/basics/installextensions.md]: Typo `Ein`

File: src/basics/installextensions.md [hunk 3]
```diff
@@ -15,34 +19,30 @@ werden und im Rahmen eines Release-Prozesses bereitgestellt werden. Darüberhina
...
 Führen Sie die folgenden Schritte aus, um diese Erweiterung zu installieren:
 
 1. Stoppen Sie den Geoserver durch einen Doppelklick auf **Stop GeoServer** im
    Ordner **Web Services** auf Ihrem Desktop.
-2. Laden Sie die WPS-Erweiterung von hier {{ book.workshopUrl }}materials/geoserver-{{ book.geoServerVersion }}-wps-plugin.zip
+1. Laden Sie die WPS-Erweiterung von hier {{ book.workshopUrl }}materials/geoserver-{{ book.geoServerVersion }}-wps-plugin.zip
    (oder der offiziellen Release-Seite) herunter. Wählen Sie bitte *Datei speichern*
    und nicht *Öffnen mit*!
-3. Wechseln Sie im Terminal in das im vorigen [Abschnitt](./folderstructure.md)
+1. Wechseln Sie im Terminal in das im vorigen [Abschnitt](./folderstructure.md)
```
Comment (annarieger) [inline src/basics/installextensions.md:38]: Ẁhy do you use always stay by 1 on your  step counter?

File: src/rest/create.md [hunk 4]
```diff
@@ -198,11 +210,12 @@ War der Befehl erfolgreich, enthält die Antwort im Terminal den Teilstring `< H
 ## Layer anlegen
 
 Der nächste logische Schritt ist das Anlegen eines Layers auf Basis einer Geometrietabelle
-aus unserem neuen Datenspeichers natural\_earth. Als Beispiel werden wir die Tabelle
+aus unserem zuvor angelegten Datenspeichers natural\_earth. Als Beispiel werden wir die Tabelle
```
Comment (annarieger) [inline src/rest/create.md]: Typo `Datenspeicher`

File: src/troubleshooting/gui.md [hunk 5]
```diff
@@ -32,8 +32,8 @@ konfigurieren. Wir wollen in diesem Abschnitt kurz auf hilfreiche Optionen hinwe
   im Bereich *Daten -\> Layer -\> Layer hinzufügen* eine Vektor-Datenbankquelle
   auswählen und dort *SQL View konfigurieren* klicken. Nützlich ist insbesondere
   die Möglichkeit die SQL-Views zu parametrisieren, etwa um bestimmte Features
-   durch entsprechende Parameter herauszufilter. Ausführliche Informationen
-   dazu finden Sie [hier](http://docs.geoserver.org/stable/en/user/data/database/sqlview.html).
+  durch entsprechende Parameter herauszufilter. Ausführliche Informationen
```
Comment (annarieger) [inline src/troubleshooting/gui.md]: Typo `herauszufiltern`

**Comment by annarieger** [comment]
Some typos found.. Otherwise +1

File: src/basics/installextensions.md (hunk 3 above)
Comment (ahennr) [inline src/basics/installextensions.md:38]: In Github Markdown these numbers don't matter. Subsequent lines starting with `1.` are formatted as numbered list and the numeration will be set automatically.

Comment (annarieger) [inline src/basics/installextensions.md:38]: Thanks for the explanation :+1:

**Comment by buehner** [comment]
Thank you very much for this enhancement @ahennr and for the review @annarieger 

I'll merge now. :+1:

**Pull request merged**